liquibase-standard/src/main/java/liquibase/database/core/MySQLDatabase.java

```diff
@@ -678,4 +678,9 @@ public boolean supportsCreateIfNotExists(Class<? extends DatabaseObject> type) {
     public boolean supportsDatabaseChangeLogHistory() {
         return true;
     }
+
+    public boolean getUseAffectedRows() throws DatabaseException {
```

Comment:
@mpvvliet could you add the `@Override` annotation here? codeql is complaining about it.

Reply:
I think you are referring to an old codeql scan, this method method does not override any method. 🤔

Reply:
haha true, it's gone now. Sorry!